code/game/machinery/bots/medbot.dm

```diff
@@ -309,7 +310,7 @@
 			stunned = 0
 		return
 
-	if(frustration > 8)
+	if(frustration >= 8)
```

Comment:
To be consistent with https://github.com/HippieStationCode/HippieStation13/blob/master/code/game/machinery/bots/secbot.dm#L211